include/pingcap/kv/Cluster.h

```diff
@@ -27,10 +29,17 @@ struct Cluster
 
     ::kvrpcpb::APIVersion api_version = ::kvrpcpb::APIVersion::V1;
 
+    std::unique_ptr<pingcap::common::FixedThreadPool> thread_pool;
+    std::unique_ptr<common::MPPProber> mpp_prober;
+
     Cluster()
         : pd_client(std::make_shared<pd::MockPDClient>())
         , rpc_client(std::make_unique<RpcClient>())
-    {}
+        , thread_pool(std::make_unique<pingcap::common::FixedThreadPool>(1))
```

Comment:
Fow now, one thread is enough. We may refresh region cache in background later, at that time, we may need multiple threads.